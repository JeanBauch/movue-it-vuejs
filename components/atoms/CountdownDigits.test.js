import { mount } from '@vue/test-utils';

import CountdownDigits from './CountdownDigits.vue';

describe('Components: Atoms: CountdownDigits', () => {
	describe('Snapshots', () => {
		it('Should match snapshot', () => {
			const wrapper = mount(CountdownDigits, {
				propsData: {
					digits: 10,
				},
			});

			expect(wrapper.html()).toMatchSnapshot();
		});
	});
});
