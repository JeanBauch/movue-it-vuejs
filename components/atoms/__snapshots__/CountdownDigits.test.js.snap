// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components: Atoms: CountdownDigits Snapshots Should match snapshot 1`] = `
"<p class=\\"w-1/2 flex gap-x-1\\"><span class=\\"w-1/2 bg-white rounded-md flex items-center justify-center py-3\\">
      1
    </span><span class=\\"w-1/2 bg-white rounded-md flex items-center justify-center py-3\\">
      0
    </span></p>"
`;
